### ISSUE TITLE
Clerks newer version is not working with react but working with older version v4.18.0

### ISSUE THREAD

**Issue by Rohankajale** [clerkinc/javascript#1340]
- [ ] Review the documentation: https://clerk.com/docs
- [ ] Search for existing issues: https://github.com/clerkinc/javascript/issues
- [ ] Go through package changelog files.
- [ ] Provide the Frontend API key from your application dashboard.

## Package + Version

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`
- [ ] other:

### Dependencies + versions

Provide a json with the dependencies used in your project (copy paste from yarn.lock / package-lock.json) or a github project / template that reproduces the issue.

**Include the @clerk/* packages and their versions!**

Example:

```json
{  
  "dependencies": {
    "react": "18.2.0"
  },
  "devDependencies": {
    "@clerk/types": "3.31.0",
    "@clerk/clerk-react": "4.13.0"
  },
  "peerDependencies": {
    "react": "18.2.0"
  }
}

```

### Browser/OS

If applicable e.g. `Chrome {version}`, `Node {version}` , `Bun {version}` , `Cloudflare worker {version}`

## Description

Describe your issue in detail, a demo repository or a sandbox would be great.

**Comment by nikosdouvlis** [comment]
Hello @Rohankajale , thanks for the report. 
This should be fixed in `4.20.0`, so please try installing `@clerk/clerk-react@4.20.0`. If you still face issues, feel free to reopen this one.

**Comment by clerk-cookie** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
